pkg/backend/httpstate/backend.go

```diff
@@ -189,11 +189,12 @@ type Backend interface {
 }
 
 type cloudBackend struct {
-	d            diag.Sink
-	url          string
-	client       *client.Client
-	escClient    esc_client.Client
-	capabilities *promise.Promise[apitype.Capabilities]
+	d                        diag.Sink
+	url                      string
+	client                   *client.Client
+	escClient                esc_client.Client
+	capabilities             *promise.Promise[apitype.Capabilities]
+	copilotEnabledForProject map[tokens.PackageName]bool
```

Comment:
Why does it need to be a map? When would it have more than a single value? Isn't the CLI operating in the context of a single project?

Reply:
This is a great question!

I do not know if an instance of the backend is ever used for multiple operations. There is another var in the block called `currentProject` and there are helpers to change it.

To stay aligned with that we could go for a `copilotEnabledForCurrentProject` and reset it to empty in `SetCurrentProject`.

Reply:
I will update to `copilotEnabledForCurrentProject`.